addons/vehicles/functions/fnc_autoThrottle.sqf

```diff
@@ -60,6 +71,10 @@ GVAR(speedLimit) = (((velocityModelSpace _vehicle) select 1) * 3.6) max 5;
         GVAR(isSpeedLimiter) = false;
     };
 
+    if ("" != call CBA_fnc_getActiveFeatureCamera) then {
```

Comment:
```suggestion
    if (call CBA_fnc_getActiveFeatureCamera != "") then {
```
I'm fine if this is ignored if others prefer it I guess, but this looks really weird to me